apps/portal/src/pages/_app.tsx

```diff
@@ -11,6 +11,7 @@ import AppShell from '~/components/global/AppShell';
 
 import type { AppRouter } from '~/server/router';
 
+import '@tih/ui/styles.css';
```

Comment:
Do we need this? It looks ok on main branch for me rn

Reply:
Gotcha, removed

Reply:
We don't need this anymore, I've handled this in 1f640fda